src/usr/local/buildpack/tools/v2/helmfile.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+function check_tool_requirements () {
+  check_semver "$TOOL_VERSION" "all"
+}
+
+function install_tool () {
+  local versioned_tool_path
+  local file
+  local arch=linux_amd64
+  local helmfile_file="${TOOL_NAME}_${TOOL_VERSION#v}_${arch}.tar.gz"
```

Comment:
```suggestion
  local helmfile_file="${TOOL_NAME}_${TOOL_VERSION}_${arch}.tar.gz"
```
`v` is already trimmed